int32 num
string labels
string[10] names
string scores
int32[10] x_offset
int32[10] y_offset
int32[10] width
int32[10] height
